long_read_seq/Unicycler_ICA.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+##############################################
+# written by Erin Young                      #
+# for uploading samples to ICA for unicycler #
+# updated : March 23, 2023                   #
+##############################################
+
+USAGE="""
+To put the appropriate nanopore and illumina files
+into ICA to run Unicycler on them. Requires a run
+name which is used as the upload destination on ICA
+as well as the sample_sheet that is used for 
+Donut Falls (a csv file with columns for the id,
+nanopore, and both fastq files for a paired run: 
+sample,fastq,fastq_1,fastq_2)
+
+The expected run time in ~2 hours  
+
+# to download from github
+Unicycler_ICA.sh run sample_sheet.csv
```

Comment:
I would use the example that you use in the README. ```bash long_read_seq/Unicycler_ICA.sh UT-GXB02179-230317 sample_sheet.csv``` I was a confused about the "run" variable/parameter till I saw the read me.